Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -284,7 +284,7 @@ private void GameTimer()
 
                     if (secondsElapsed++ == 30 && Players.Any(x => !notifiedPlayers.Contains(x.Id))) //every 30 seconds, tell in group who have joined
                     {
-                        SendWithQueue(GetLocaleString("HaveJoined", Players.Where(x => !notifiedPlayers.Contains(x.Id)).Aggregate("", (cur, p) => cur + p.GetName() + ", ").TrimEnd(',', ' ')));
+                        SendWithQueue(GetLocaleString("HaveJoined", Players.Where(x => !notifiedPlayers.Contains(x.Id)).Aggregate("", (cur, p) => cur + p.GetName() + §" (ID: {p.Id}), ").TrimEnd(',', ' ')));
```

Comment:
Also, p.Id is not the player's Telegram id... should be p.TeleUser.Id or something similar